packages/cashier/src/pages/withdrawal/crypto-withdraw-receipt-wallet/crypto-withdraw-receipt-wallet.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { observer, useStore } from '@deriv/stores';
+import { Div100vhContainer, Icon, Text, WalletCard, Clipboard, Button } from '@deriv/components';
+import { useCashierStore } from '../../../stores/useCashierStores';
+import './crypto-withdraw-receipt-wallet.scss';
+import { Localize, localize } from '@deriv/translations';
+import { useActiveWallet } from '@deriv/hooks';
+
+const CryptoWithdrawReceiptWallet = observer(() => {
+    const active_wallet = useActiveWallet();
```

Comment:
@hamid-deriv These are wallet-related files, Why we are putting them in the cashier? 🤔 I saw there is an `is_wallet` flag added to the cashier to handle things conditionally 🤔

Reply:
@farzin-deriv Because we are handling these components from the cashier package we need to have them here, otherwise we have to import components from the appstore in the cashier.
Also, we are using the same component from the cashier for the wallet but there are some differences. For handling these changes we use `is_wallet` prop. We can not separate components for these changes because these changes are minor and it's just duplicating code if we separate them.
When the design of the wallet and cashier become the same, most of these prop usages will be removed.

Reply:
@farzin-deriv We are using the stores from `cashier` and `cfd` packages in the appstore and vice versa. We need to consider these in the spliting wallets from appstore  project .